pam/adapter/return.go

```diff
@@ -1,49 +1,59 @@
-package main
+package adapter
 
 import (
 	"github.com/msteinert/pam/v2"
 )
 
 // Various signalling return messaging to PAM.
 
-// pamReturnStatus is the interface that all PAM return types should implement.
-type pamReturnStatus interface {
+// PamReturnStatus is the interface that all PAM return types should implement.
+type PamReturnStatus interface {
 	Message() string
 }
 
-// pamReturnError is an interface that PAM errors return types should implement.
-type pamReturnError interface {
-	pamReturnStatus
+// PamReturnError is an interface that PAM errors return types should implement.
+type PamReturnError interface {
+	PamReturnStatus
 	Status() pam.Error
 }
 
-// pamSuccess signals PAM module to return with provided pam.Success and Quit tea.Model.
-type pamSuccess struct {
+// PamSuccess signals PAM module to return with provided pam.Success and Quit tea.Model.
+type PamSuccess struct {
 	brokerID string
 	msg      string
 }
 
 // Message returns the message that should be sent to pam as info message.
-func (p pamSuccess) Message() string {
+func (p PamSuccess) Message() string {
 	return p.msg
 }
 
-// pamIgnore signals PAM module to return pam.Ignore and Quit tea.Model.
-type pamIgnore struct {
+// BrokerID returns the broker id that authenticated the user.
+func (p PamSuccess) BrokerID() string {
```

Comment:
Why not just exporting BrokerID and Msg? rather than having 2 accessors?

Reply:
Ah, well... I thought was "cleaner", in the sense, that's _public to read_ and _private to write_... Probably not fully idiomatic, not sure :)